### ISSUE TITLE
snet-cli 1.8 installed from PyPi cannot find /usr/local/lib/python3.6/dist-packages/snet_cli/resources/proto

### ISSUE THREAD

**Issue by vsbogd** [singnet/snet-cli#146]
snet-cli version 
```sh
root@cee513519bc4:/singnet/src/github.com/singnet/examples/02-grpc-service# snet version  
    version: 0.1.8

root@cee513519bc4:/singnet/src/github.com/singnet/examples/02-grpc-service# snet --print-traceback client call 0 3 localhost:8080 echo '{"message": "hello"}'
/usr/local/lib/python3.6/dist-packages/snet_cli/resources/proto: warning: directory does not exist.
state_service.proto: No such file or directory
Traceback (most recent call last):
  File "/usr/local/bin/snet", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.6/dist-packages/snet_cli/__init__.py", line 20, in main
    getattr(args.cmd(conf, args), args.fn)()
  File "/usr/local/lib/python3.6/dist-packages/snet_cli/mpe_client_command.py", line 379, in call_server_statelessly
    current_nonce, current_amount, unspent_amount = self._get_channel_state_statelessly(grpc_channel, self.args.channel_id)
  File "/usr/local/lib/python3.6/dist-packages/snet_cli/mpe_client_command.py", line 350, in _get_channel_state_statelessly
    server     = self._get_channel_state_from_server    (grpc_channel, channel_id)
  File "/usr/local/lib/python3.6/dist-packages/snet_cli/mpe_client_command.py", line 327, in _get_channel_state_from_server
    stub_class, request_class, _ = import_protobuf_from_dir(codegen_dir, "GetChannelState")
  File "/usr/local/lib/python3.6/dist-packages/snet_cli/utils_proto.py", line 22, in import_protobuf_from_dir
    raise Exception("Error while loading protobuf. Cannot find method=%s"%method_name)
Exception: Error while loading protobuf. Cannot find method=GetChannelState
```

**Comment by astroseger** [comment]
@vforvalerio87 Could you include snet_cli/resources/proto in pip distribution?

**Comment by vforvalerio87** [comment]
That's a bug with the MANIFEST.in:
```
include snet_cli/resources/contracts/abi/*
include snet_cli/resources/contracts/networks/*
include snet_cli/_vendor/*
```

It's missing `snet_cli/resources/proto/*`. I'll make a PR and submit it, then I propose we release the hotfix as 1.8.1

**Comment by astroseger** [comment]
But 1.8.0 will still be broken... I would just replace it without  version increment. In any case we will have new version in (I hope) less then one week (with new Registry).

**Comment by vforvalerio87** [comment]
It cannot be done (anymore) and it's not a good practice (https://mail.python.org/pipermail/distutils-sig/2015-January/025683.html) because it used to break the package manager caching mechanisms, forcing people to reinstall packages with the same name. Releasing a new version is the only option really

**Comment by vforvalerio87** [comment]
Fixed in #147
